docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,892 @@
+    "    f\"The obtained solution places a partition between nodes {maxcut_partition[0]} \"\n",
+    "    f\"and nodes {maxcut_partition[1]}.\"\n",
+    ")\n",
+    "maxcut.draw(results, pos=nx.spring_layout(graph, seed=seed))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Inspect the results of subroutines"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `MinimumEigensolverResult` ([details](https://qiskit.org/documentation/stubs/qiskit.algorithms.MinimumEigensolverResult.html)) that results from performing VQE on the relaxed Hamiltonian is available:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<qiskit_algorithms.minimum_eigensolvers.vqe.VQEResult at 0x179302fa0>"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results.relaxed_result"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result of the rounding scheme is also worth considering. In this example, we used the `SemideterministricRounding`. It's important to note that with semi-deterministic rounding, a single sample is generated as the result, making it the optimal solution candidate.\n",
+    "\n",
+    "However, if we use the `MagicRounding` instead, multiple samples would be generated, each with a probability associated with it. These probabilities sum up to one, providing a distribution of potential optimal solutions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[SolutionSample(x=array([1, 0, 1, 1, 0, 1]), fval=6.0, probability=1.0, status=<OptimizationResultStatus.SUCCESS: 0>)]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results.samples"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exact Problem Solution with the `NumpyMinimumEigensolver`\n",
+    "\n",
+    "To assess the performance of QRAO in approximating the optimal solution, we can utilize the `NumpyMinimumEigensolver`, an exact classical optimizer. We can obtain the exact optimal solution to the problem as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "objective function value: 9.0\n",
+      "variable values: x_0=0.0, x_1=1.0, x_2=0.0, x_3=1.0, x_4=1.0, x_5=0.0\n",
+      "status: SUCCESS\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.algorithms.minimum_eigensolvers import NumPyMinimumEigensolver\n",
```

Comment:
```suggestion
    "from qiskit_algorithms import NumPyMinimumEigensolver\n",
```
Happy it just works with that one too, but we should be using qiskit_algorithms one